packages/aggregator/src/__tests__/Aggregator/Aggregator.test.js

```diff
@@ -40,7 +40,7 @@ const aggregationOptions = {
   aggregations: [
     {
       type: 'MOST_RECENT',
-      config: { orgUnitToGroupKeys: [], requestedPeriod: '20200214;20200215' },
+      config: { orgUnitMap: {}, requestedPeriod: '20200214;20200215' },
```

Comment:
Bit of a tangential change, cleaning up a couple of places that used a different key name. `orgUnitMap` might not be ideal, but at least it's consistent